src/components/Menu.vue

```diff
@@ -29,7 +29,10 @@
             <p class="overline">{{ instanceUrl }}</p>
             {{ eComStore.storeName }}
           </ion-label>
-          <ion-note slot="end">{{ userProfile?.userTimeZone }}</ion-note>
+          <div>
+            <ion-note slot="end">{{ userProfile?.userTimeZone }}</ion-note>    
+            <ion-note slot="end">{{ currentShopifyConfigId }}</ion-note>
```

Comment:
we should put this as a p tag below the ion label instead of putting it in the ion note